AnnotationModule.py

```diff
@@ -0,0 +1,63 @@
+import csv
+import xml.etree.ElementTree as ET
+import cv2
+import numpy as np
+import openslide
+import os
+from skimage import io
+from BaseImage import printMaskHelper
+from PIL import Image, ImageDraw
+
+def getPoints(xmlfile):
+    # create element tree object

```

Comment:
try to think about other users. is it obvious from your code what this XML file should contain? what format it should be in? what if someone wants to write their own XML file programmatically and have it ingested by your module, should they try and reverse engineer it from your parser?